routers/article-routers.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const router = express.Router();
+const {
+  getArticleById,
+  handleInvalidMethod,
+} = require("../controllers/article-controllers");
+
+router
+  .route("/api/articles/:article_id")
```

Comment:
any reason why you are using express router ?  Can't see how it makes code any cleaner.  though can't see any drawback yet